src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/EditMetadataByCreatingNewTypeAttribute.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>
+    /// Reserved to be used by the compiler to emit new types
+    /// rather than updating them when applying metadata updates
+    /// </summary>
+
```

Comment:
```suggestion
    /// <summary>
    /// Indicates a type should be replaced rather than updated when applying metadata updates.
    /// </summary>
```